src/python/pants/help/help_info_extracter_test.py

```diff
@@ -373,14 +371,14 @@ def fake_consumed_scopes_mapper(scope: str) -> Tuple[str, ...]:
                             "ranked_values": ({"rank": Rank.NONE, "value": None, "details": None},),
                         },
                         "typ": str,
-                        "default": None,
+                        "default": ["a", "b", "c"],
                         "help": "No help available.",
                         "deprecation_active": False,
                         "deprecated_message": None,
                         "removal_version": None,
                         "removal_hint": None,
-                        "choices": ("a", "b", "c"),
-                        "comma_separated_choices": "a, b, c",
+                        "choices": None,
+                        "comma_separated_choices": None,
```

Comment:
None of our new types support this, so I nixed it from the test, and it's ripe to be nixed altogether.

I assume the spiritual successor is enum options.